### PR TITLE
pulsar-3 config

### PR DESCRIPTION
**Pull request by levackt**
Secret Network's testnet is being updated from pulsar-2 to pulsar-3, see https://docs.scrt.network/secret-network-documentation/development/resources-api-contract-addresses/connecting-to-the-network/testnet-pulsar-3

I've tried to add a new chain by using the experimental suggest chain with this page https://connect.pulsar3.scrttestnet.com/

It then connects but doesn't work. 
I can also edit an existing pulsar-2 config to edit the endpoints, I can see the balances but I'm unable to send. 
I assume it's because it's still using the chain-id pulsar-2.

Let me know if this PR is OK, or how I can help with the transition.

### CODE REVIEW

File: src/index.ts [hunk 1]
```diff
@@ -71,6 +71,7 @@ const main = async () => {
         "osmo-test",
         "pion",
         "theta-testnet",
+        "pulsar",
```
Comment (HeesungB) [inline src/index.ts:74]: @levackt Could you remove this value?

**Comment by HeesungB** [comment]
@levackt Could you add `Node Provider` field?
The chain to add the test net to the `index.ts` file is only added when you are operating a node directly from `Keplr Infra.

**Comment by HeesungB** [comment]
@levackt Could you remove `coinGeckoId` in currency, stakeCurrency, feeCurrency?
Mainnet and testnet tokens can be confusing.

**Comment by HeesungB** [comment]
@levackt Thank you for contribution!

**Comment by levackt** [comment]
@HeesungB
Apologies, this PR was opened sooner than expected.
There's a new PR to revert and hope you don't mind, we'll re-open in a few
days again when Devs have had time to migrate.
Sorry again, thanks for the assistance.

**Pull request merged**